sdk/core/azure-core/inc/azure/core/internal/json/json.hpp

```diff
@@ -2176,7 +2176,7 @@ _az_JSON_HEDLEY_DIAGNOSTIC_PUSH
     class exception : public std::exception {
     public:
       /// returns the explanatory string
-      _az_JSON_HEDLEY_RETURNS_NON_NULL const char* what() const noexcept override
+      const char* what() const noexcept override
```

Comment:
Does this line needs to be modified?

Reply:
I see, went to the link you possted.

Reply:
just added a comment in the description.

This is from https://github.com/Azure/azure-sdk-for-cpp/pull/2677   which also affect us when using /analyze.  I am just bringing a patch from its repo